README.md

```diff
@@ -1 +1,49 @@
-# ros2_generate_interface_docs
\ No newline at end of file
+# ros2_generate_interface_docs
+
+This package will generate the static HTML documentation for messages, services and actions.
+It will create the documentation for all the available interfaces in your ROS2 workspace.
```

Comment:
Why should this be tied to a ROS workspace at all and not take arguments pointing to e.g. packages?

Reply:
Also how does it handle if interface files are being generated at build time by a package?

Reply:
I created a ticket for your first comment https://github.com/ros2/ros2_generate_interface_docs/issues/2

can you give some pointers to a interface generated at build time ?